coral/types/api.d.ts

```diff
@@ -3091,8 +3091,8 @@ export type components = {
             includeOnlyFailedTasks: boolean;
         };
         ChangePasswordRequestModel: {
-            pwd: string;
-            repeatPwd: string;
+            pwd?: string;
```

Comment:
[inquiry]
for a password change model, the passwords should be required, even if the initial password can be blank for other technical reasons, password reset must always receive a value, wdyt?

Reply:
It should not be the case. Fixed.